source/desktop/sharedexchange/sharedexdns.md

```diff
@@ -9,22 +9,27 @@ If you would like to set these up manually please find our guide here, or contac
 
 Please allow time for any DNS changes to propagate through the internet, this may take up to 24 hours. Once complete, you are ready to use your email client to connect to your new mailbox.
 
-
-## If you are not using UKFast's SafeDNS system  
+## If you are not using UKFast's SafeDNS system
 
 You should update your DNS records with your DNS provider, the MX records should be set as:
 
-
-![DNS Records](files/dns/sharedexchangedns.PNG)
-
+| **Mail Exchanger** | **Priority** |
+| -------------- | -------- |
+| `hub1.ukfastexchange.co.uk` | `10` |
+| `hub2.ukfastexchange.co.uk` | `10` |
+| `hub3.ukfastexchange.co.uk` | `10` |
+| `hub4.ukfastexchange.co.uk` | `10` |
+| `client.ukfastexchange.co.uk` | `20` |
 
 Next, you should add an SRV record to the domain to allow for auto-discovery. This allows your Exchange ready client, such as Outlook, to find all the settings it needs to connect to our shared Exchange platform.
 
 Here is the information you need for the SRV record, in its raw format, should you not be using our UKFast SafeDNS system for DNS:
 
 `_autodiscover._tcp.DOMAIN.COM. 0 0 443 client.ukfastexchange.co.uk`
 
+```
```

Comment:
The note needs to be like this
```
``` note:: you must change “DOMAIN.COM” with your own domain name.
```
with closing backticks which i apparently can't show here.

Reply:
Resolved in f84ee4f